compose/cli/main.py

```diff
@@ -280,7 +282,10 @@ def ps(self, project, options):
                     container.human_readable_state,
                     container.human_readable_ports,
                 ])
-            print(Formatter().table(headers, rows))
+            if (options['--json']):
+                print(json.dumps(rows))
```

Comment:
The json output should be a json object so that each field has a key. I believe with this formatting it would be just an array of fields without any keys.